[Docs] `html`: shallow renders full tree (#1912)
airbnb/enzyme: docs/api/ShallowWrapper/html.md

# `.html() => String`

> Returns a string of the rendered HTML markup of the current render tree. The full tree is rendered, not only the Shallow part.

Reviewer: ```suggestion
Returns a string of the rendered HTML markup of the entire current render tree (not just the shallow-rendered part).
```
